app/jobs/appointment_reminders_job.rb

```diff
@@ -0,0 +1,28 @@
+class AppointmentRemindersJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Appointment.within_48_hours.each do |appointment|
+      send_reminder(appointment)
```

Comment:
I wonder if it makes sense for this job to fan out individual jobs per appointment. What happens if one (or more) fails here?